_tests/_DebugEnv.test.js

```diff
@@ -0,0 +1,8 @@
+import env from './_env'
+
+describe('DebugEnv', () => {
+  it('should output the environment that Jest is testing', () => {
+    console.log('E2E baseUrl: ', env.baseUrl);
+    expect(true).toBe(true);
+  })
+});
```

Comment:
Previously we weren't sure if any test will always be run, so I added a specific always-passing test to output the URL and reduce the repetition in Travis output.